modules/front-end/src/app/core/components/project-drawer/project-drawer.component.ts

```diff
@@ -80,7 +80,7 @@ export class ProjectDrawerComponent implements OnInit {
     if (!this.isEditing) { // creation
       return this.permissionsService.isGranted(generalResourceRNPattern.project, permissionActions.CreateProject);
     } else {
-      const rn = this.permissionsService.getResourceRN(ResourceTypeEnum.Project, this.project);
+      const rn = this.permissionsService.getProjectRN(this.project.name);
```

Comment:
I recommend pass the whole project object to `getProjectRN` function.

Reply:
I would like to pass only the info needed